### PR TITLE
describe how to contribute using the dockerfile used in production

### PR DESCRIPTION
**Pull request by petrus-v**
... and make it possible

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -98,10 +98,70 @@ create`` command, by inspecting the location of the created volumes with
...
+or using and testing the docker image (with python >= 3.5)::
+
+    $ docker build -t anybox/buttervolume docker/
+    $ sudo docker run -it --rm --privileged \
+        -v /var/lib/docker:/var/lib/docker \
+        -v "$PWD":/usr/src/buttervolume \
+        -w /usr/src/buttervolume: \
+        anybox/buttervolume test
+
+If you haven't a BTRFS partition or volume you can setup a virtual partition
```
Comment (ccomb) [inline README.rst]: haven't a → have no

File: README.rst [hunk 2]
```diff
@@ -98,10 +98,70 @@ create`` command, by inspecting the location of the created volumes with
...
+in a file as follow (tested on Debian 8):
```
Comment (ccomb) [inline README.rst]: as follows

File: README.rst [hunk 3]
```diff
@@ -98,10 +98,70 @@ create`` command, by inspecting the location of the created volumes with
...
+    Turning ON incompat feature 'extref': increased hardlink limit per file to 65536
+    ERROR: device scan failed '/var/lib/docker/btrfs.img' - Block device required
+    fs created label (null) on /var/lib/docker/btrfs.img
+        nodesize 16384 leafsize 16384 sectorsize 4096 size 10.00GiB
+
+.. note::
+
+   you can ignore errors error, in fact the new FS is formated
+
+* Mount the partition somewhere temporarely to create 3 new btrfs subvolumes::
```
Comment (ccomb) [inline README.rst]: temporarily

File: docker/entrypoint.sh [hunk 4]
```diff
@@ -1,3 +1,21 @@
 #!/bin/bash
+
+SSH_PORT=${SSH_PORT:-1122}
+
+echo "Port 1122" >> /etc/ssh/sshd_config
```
Comment (ccomb) [inline docker/entrypoint.sh]: File /etc/ssh/sshd_config  may grow up with a lot of "Port 1122" after multiple restart ?

**Comment by ccomb** [comment]
apart from the few comments, it lgtm

File: docker/entrypoint.sh (hunk 4 above)
Comment (petrus-v) [inline docker/entrypoint.sh]: You're right!

Thanks for reviews

**Comment by petrus-v** [comment]
last commit should make the entry point idempotent !

**Pull request merged**